meta.go

```diff
@@ -0,0 +1,14 @@
+package azuretpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "azure.cluster.giantswarm.io"
```

Comment:
Yes `azure-cluster`. Then you do `kubectl get azureclusters` instead `kubectl get azures`.